src/implementations/twig/components/card/__snapshots__/card.test.js.snap

```diff
@@ -273,7 +273,7 @@ exports[`Card Default renders correctly 1`] = `
               Lorem ipsum dolor sit amet, 
               <a
                 class="ecl-link"
-                href="#"
+                href="/example"
```

Comment:
the random part is not applied here

Reply:
this is a snapshot, the alteration of the link only happens in storybook...